src/main/java/com/redhat/red/build/koji/model/xmlrpc/KojiTaskInfo.java

```diff
@@ -226,4 +242,81 @@ public void setRequest( List<Object> request )
     {
         this.request = request;
     }
+
+    @Override
+    public boolean equals( Object o )
```

Comment:
I added equals() and hashCode() for this class modeled off of the other classes.